CONTRIBUTING.md

```diff
@@ -576,12 +613,16 @@ to define the postgres chart that Concourse depends on.
 - `CONCOURSE_CHART_DIR`: location in the filesystem where a copy of [`the Concourse Helm
 chart`][concourse-helm-chart] exists.
 
+
 With those set, go to `topgun/k8s` and run Ginkgo:
 
 ```sh
 ginkgo .
```

Comment:
wdyt @taylorsilva @aoldershaw ?

```suggestion
# run the test cases serially
ginkgo .

# run the test cases with a concurrency level of 16
ginkgo -nodes=16 .
```

Reply:
I like it!

Reply:
206c85db503b6d332952516fb953ccea175812cd